designs/index.md

```diff
@@ -14,27 +14,37 @@ changes as designs are implemented.
 Always go to the [documentation]({{ site.docs_site_url }}) for descriptions of
 current Bazel functionality.
 
-## Table of Contents
-{:.no_toc}
-
-* ToC
-{:toc}
-
-
 ## When do I need a design document and review?
 
 If you're planning to add, change, or remove a user-facing feature, or make a
-significant architectural change to Bazel, you **must** write a design document
-and have it reviewed before you can submit the change. See [What is a
-significant change?](#what-is-a-significant-change) for details.
+*significant architectural change* to Bazel, you **must** write a design document
+and have it reviewed before you can submit the change.
 
 Implementation can begin before the proposal is accepted, for example as a
 proof-of-concept or an experimentation. However, you cannot submit the change
 before the review is complete.
 
+## What is a significant change?
 
-## Workflow
+There are no hard and fast criteria, but here are some examples:
+
+*   Addition or deletion of native build rules
+*   Breaking-changes to native rules
+*   Changes to a native build rule semantics that affect the behavior of more
+    than a single rule
+*   Changes to Bazel's rule definition API
+*   Changes to the APIs that Bazel uses to connect to other systems
+*   Changes to the Starlark language, semantics, or APIs
+*   Changes that could have a pervasive effect on Bazel performance or memory
+    usage (for better or for worse)
+*   Changes to widely used internal APIs
+*   Changes to flags and command-line interface.
 
+When a proposal adds, removes, or modifies any function or object available in
+BUILD, WORKSPACE, or bzl files, the [Starlark team](../maintainers-guide.html)
+has to be in the reviewers list.
+
+## Workflow
 
 ### Write the design document
 
```

Comment:
The list in the header doesn't quite match the docs template. Which should change? I've added a few suggestions, but this list is also in a different order.

Reply:
Changed headers